build/yamls/base/crds.yml

```diff
@@ -27,6 +31,10 @@ spec:
     - name: v1beta1
       served: true
       storage: true
+      schema:
```

Comment:
Is the a new field that is not present in v1beta1? want to check if a cluster that already uses antrea can upgrade to 0.10 by simply fixing `apiextensions.k8s.io/v1beta1`, or they need to fix all the CRD definition.

Reply:
they do need to update the definition.. as there is some renaming "JSONPath -> jsonPath" and some fields no longer supported (preserveUnknownFields) and some fields have a different parent field (validation -> schema)